memory.x

```diff
@@ -1,6 +1,6 @@
 /* Linker script for the STM32F103C8T6 */
 MEMORY
 {
-  FLASH : ORIGIN = 0x08000000, LENGTH = 128K
+  FLASH : ORIGIN = 0x08000000, LENGTH = 64K
```

Comment:
Is there a reason to change this?

Reply:
Fwiw independantly ran into a note that there are some variants of the stm32f103 on some blue pills that do have 128K ; so i wonder if @jamwaffles has one of those ;) .. the STM32F103C8T6 officially only has 64K though

Reply:
That's exactly it! I don't think I own a single non-bootleg F103 :grin: 

FWIW this setup is way out of date. I believe these days one doesn't need to provide their own `memory.x`, so this file could probably be deleted.

Reply:
not with stm32f1xx-hal afaik; i'm prepping a follow-up PR that moves the examples to embassy for the pac (as that implements hal 1.x) together with the patches from #207 so that that can be merged hopefully.. That will also drop memory.x